scripts/changelog/templates/host_functions.md.tera

```diff
@@ -0,0 +1,30 @@
+{%- import "change.md.tera" as m_c -%}
+{%- set_global host_fn_count = 0 -%}
+
+## Host functions
+
+{% for pr in changes | sort(attribute="merged_at") -%}
+
+{%- if pr.meta.B and pr.meta.B.value == 0 -%}
+{#- We skip silent ones -#}
+{%- else -%}
+    {%- if pr.meta.E and pr.meta.E.value == 4 -%}
+        {%- set_global host_fn_count = host_fn_count + 1 -%}
+            - {{ m_c::change(c=pr) }}
+        {% endif -%}
+    {% endif -%}
+{%- endfor -%}
+
+<!-- {{ host_fn_count }} host functions were detected -->
+
+{% if host_fn_count == 0 -%}
+ℹ️ This release does not contain any new host functions.
+{% elif host_fn_count == 1 -%}
+⚠️ The runtimes in this release contain one new **host function**.
```

Comment:
I would skip the whole section if none were detected

Reply:
Sure, I will leave us a comment so we know it was just not skipped for no reason but I can hide this to the user.

Reply:
Hmm this is actually what I initially went for initially but I currently do not know the count ahead of time.
So I run thru the list, keep counting and display the list, thne comes this message.
So if I display nothing that will look like a weird section with no content.

I agree with your goal but the tooling currently does not allow it.

Reply:
ok, it's trivial to delete manually, I'd say